test/Elastic.Apm.Feature.Tests/CloudProviderSteps.cs

```diff
@@ -37,7 +37,7 @@ public void GivenAnAgentConfiguredWith(Table table)
 
 			var output = _scenarioContext.ScenarioContainer.Resolve<ITestOutputHelper>();
 			var logger = new XUnitLogger(LogLevel.Trace, output);
-			var config = new MockConfiguration(logger, cloudProvider: cloudProvider, flushInterval: "0");
+			var config = new MockConfiguration(logger, cloudProvider: cloudProvider, flushInterval: "1ms");

```

Comment:
Aren't we testing a different case now?

Reply:
I'd argue the core of the test (and what we test) is the same, but the agent config we test with is different.

I saw these tests failing and it was because of this bug specifically when `flushInterval` was set to `0`. Now.. these tests here test cloud metadata collection (nothing to do with how data is sent) - so what we assert on is that we parse and report cloud metadata.

Because of the bug with `flushInterval`, these tests failed - the reason was that events weren't sent (due to the bug), so the asserts on cloud metadata failed.

Since we have good test coverage for `flushInterval` - those tests properly failed as well, I thought let's make it independent from `flushInterval`. 

But setting it to `1ms` is arbitrary. I just removed setting the `flushInterval` and let it run with default.

So the point of the change was to not set `flushInterval` to `0` because that's not the default and let's not test `flushInterval=0` as a side-effect in these tests.